pkg/tnf/testcases/data/cnf/gatherfacts.go

```diff
@@ -30,6 +30,17 @@ var GatherPodFactsJSON = string(`{
         "ALLOW_ALL"
       ]
     },
+    {
+      "name": "imagePullPolicy",
+      "skiptest": false,
+      "command": "oc get pod %s -n %s -o json  | jq -r '.spec.containers[0].imagePullPolicy'",
```

Comment:
With .spec.containers[0] you're only checking the first container of the pod. This should be either in privilegedpod.go (with containers[%d]) or in a new json-based test.